src/integration/common/SEO.js

```diff
@@ -192,5 +192,25 @@ export default () => {
         });
       });
     });
+
+    describe('Apple Touch Icon', () => {
+      const touchIconScript = document.querySelectorAll(
+        'head > script[rel="apple-touch-icon"]',
+      );
+
+      touchIconScript.forEach(script => {
+        const url = script.getAttribute('href');
+        const sizes = script.getAttribute('sizes');
+
+        it('should be in the document', () => {
+          expect(script).toBeInTheDocument();
+        });
+
+        it(`should match attributes`, () => {
+          expect(`${url}`).toMatchSnapshot();
+          expect(`${sizes}`).toMatchSnapshot();
```

Comment:
Can we maybe separate these into 2 separate assertions?

```suggestion
        it(`should match url`, () => {
            expect(`${url}`).toMatchSnapshot();
          };
          it(`should match size`, () => {
            expect(`${sizes}`).toMatchSnapshot(); 
          };
          expect(`${sizes}`).toMatchSnapshot();
```

Reply:
Ignore this - I have a slightly better suggestion :)